cloud/logger/README.md

```diff
@@ -1,18 +1,32 @@
 # RainCatcher Logger Cloud
 
-This module creates logging facade for node.js based applications that is used by all Raincatcher modules.
-Users can use their own loggers by wrapping them into provided interface.
-By default module provides [Bunyan](https://www.npmjs.com/package/bunyan) and Console loggers implementations.
+This module creates logging facade for node.js based applications that
+is used by all Raincatcher modules. Users can use their own loggers by
+wrapping them into provided interface. By default the logger is set to
+an empty logger which returns nothing. There are Two modules provide
```

Comment:
`By default the logger is set to an empty logger which returns nothing.`
Loggers prints out to stdout and always return nothing. 
Best to say that by default logging is turned off.